stdlib/3.3/lzma.pyi

```diff
@@ -0,0 +1,105 @@
+import io
+import sys
+from typing import Any, BinaryIO, IO, List, Mapping, Optional, Union
+
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _PathOrFile = Union[str, bytes, IO[Any], PathLike[Any]]
+elif sys.version_info >= (3, 3):
+    _PathOrFile = Union[str, bytes, IO[Any]]
+else:
+    _PathOrFile = str
+
+_FilterChain = List[Mapping[str, Any]]
```

Comment:
From https://docs.python.org/3/library/lzma.html#specifying-custom-filter-chains, it sounds like this can actually be any Sequence.

This is important because List is invariant, so if you specify the type like this, users are only allowed to pass lists of exactly `Mapping[str, Any]`, not lists of objects of some subclass of `Mapping[str, Any]`.

Reply:
Thanks! checking the C source these can indeed be Sequences: fc4eb12